### ISSUE TITLE
Create a dataset loader for MedQuAD

### ISSUE THREAD

**Issue by WojciechKusa** [bigscience-workshop/biomedical#169]
## Adding a Dataset
- **Name:** *MedQuAD*
- **Description:** *MedQuAD includes 47,457 medical question-answer pairs created from 12 NIH websites (e.g. cancer.gov, niddk.nih.gov, GARD, MedlinePlus Health Topics). The collection covers 37 question types (e.g. Treatment, Diagnosis, Side Effects) associated with diseases, drugs and other medical entities such as tests.*
- **Task:** *QA, RQE*
- **Paper:** *https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3119-4*
- **Data:** *https://github.com/abachaa/MedQuAD*
- **License:** *CC BY 4.0*
- **Motivation:** *Study biomedical Q&A with answers from reliable sources*

**Comment by giyaseddin** [comment]
#self-assign

**Comment by jason-fries** [comment]
Hi @giyaseddin can you let us know if you are still working on this so we can update our project board? Please just notify us the status **by Friday April 8**.  You can response to this comment or ping us on Slack or Discord. 

No worries if you are not finished but still intend to work on this!

**Comment by giyaseddin** [comment]
Yeah, I'm planning to open a PR soon, also I'm waiting in case another dataset frees up!
Thanks for asking @jason-fries
